CONTRIBUTING_TO_OPENSHIFT_PLUGINS.md

```diff
@@ -0,0 +1,280 @@
+# Development and release process for the OpenShift Jenkins plugins
+
+The OpenShift organization currently maintains for plugins related to integration between V3 OpenShift and V2 Jenkins:
+
+* [OpenShift Client Plugin](https://github.com/openshift/jenkins-client-plugin)
+* [OpenShift Sync Plugin](https://github.com/openshift/jenkins-sync-plugin)
+* [OpenShift Login Plugin](https://github.com/openshift/jenkins-openshift-login-plugin)
+* [OpenShift Pipeline Plugin (only in maintenance at this point)](https://github.com/openshift/jenkins-plugin)
+
+Per the process for [hosting plugins](https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins) specified by the open source Jenkins
+project, there are associate wikis in the Jenkins project (see the README of each plugin) and analogous repositories under the 
+`jenkinsci` project on GitHub.  Those URLs are:
+
+* [OpenShift Client Plugin](https://github.com/jenkinsci/openshift-client-plugin)
+* [OpenShift Sync Plugin](https://github.com/jenkinsci/openshift-sync-plugin)
+* [OpenShift Login Plugin](https://github.com/jenkinsci/openshift-login-plugin)
+* [OpenShift Pipeline Plugin (only in maintenance at this point)](https://github.com/jenkinsci/openshift-pipeline-plugin)
+
```

Comment:
Maybe should link to the README of each plugin instead of having another set of the same links?

Reply:
There are valid times when the versions will not be in sync, and the READMEs should be different.